shoop/themes/classic_gray/__init__.py

```diff
@@ -53,8 +65,10 @@ def get_footer_links(self):
     def get_cms_links(self):
         if "shoop.simple_cms" not in settings.INSTALLED_APPS:
             return
+
         from shoop.simple_cms.models import Page
-        for page in Page.objects.visible().filter(visible_in_menu=True):
+        page_ids = self.get_setting("footer_cms_pages") or []
```

Comment:
Since @Pikkupomo 's recent PR, this method is also used by the top navigation...